src/app/clusters/tls-client-management-server/BUILD.gn

```diff
@@ -13,6 +13,16 @@
 # limitations under the License.
 import("//build_overrides/chip.gni")
 
-group("tls-client-management-server") {
+source_set("tls-client-management-server") {
+  sources = [ "tls-client-management-server.h" ]
+
+  public_deps = [
```

Comment:
is this needed? we already have this in app_config_dependent_sources.gni, is it not enough?